packages/spatial/src/renderer/components/MeshComponent.ts

```diff
@@ -43,7 +43,7 @@ import { addObjectToGroup, removeObjectFromGroup } from './GroupComponent'
 
 export const MeshComponent = defineComponent({
   name: 'MeshComponent',
-  jsonID: 'EE_mesh',
```

Comment:
Why are we removing json id for meshes? Don't we want to be able to serialize meshes into a scene?

Reply:
Meshes are threejs objects, how would they get serialized?